n2ws/manifest.json

```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "n2ws/assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "n2ws/assets/dashboards/N2WSBackup&Recovery-GraphicalVersion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "n2ws/assets/dashboards/N2WSBackup&Recovery-GraphicalVersion-Areas.json"
```

Comment:
```suggestion
	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
```

Reply:
You need to either apply this change or rename the file to "N2WSBackup&Recovery-GraphicalVersion-Areas.json" (notice the capitalisation difference)